web-server/django_web_server/django_web_server/static/stylesheets/website/website.js

```diff
@@ -1,11 +1,97 @@
 
-var map = L.map('mapid').setView([30, 0], 0.5);
+var map = L.map('mapid').setView([60, 10], 3);
 L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
               maxZoom: 19,
               attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
                 }).addTo(map);
+
 var jsonData = JSON.parse(document.querySelector('#jsonData').getAttribute('data-json'));
 var locations = jsonData.locations;
-for (const loc of locations) {
-  L.marker(loc).addTo(map);
+var greenIcon = new L.Icon({
+  iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-green.png',
+  shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
+  iconSize: [25, 41],
```

Comment:
An improvement would be to replace the hardcoded values with constants with explanatory names.